backend/routes/courses/swap/accecptSwapCourses.js

```diff
@@ -0,0 +1,58 @@
+'use strict'
+
+const axios = require('axios')
+const urlJoin = require('url-join')
+
+const { HttpError } = require('../../../errors')
+const env = require('../../../env')
+const DB_BASE_URL = env('DB_BASE_URL')
+
+/**
+ * @typedef ErrorResponse
+ * @property {[integer]} statusCode
+ * @property {[string]} message
+ */
+
+/**
+ * @typedef SwapeeRequest
+ * @property {[string]} swapeeUserId -- the userId of the person initiating the swap
+ * @property {[string]} courseToGiveId -- the course being offered up by the swaperUser
+ * @property {[string]} courseToGetId -- the course desired by the swaperUser
+ * @property {[string]} swapeeAccept -- status of swapee
+ */
+
+/**
+ * @typedef CourseSwapResponse
+ * @property {[string]} courseSwapId -- the primary key (ID) of this swap
+ * @property {[string]} courseToGetId -- the course desired by the swaperUser
+ * @property {[string]} courseToGiveId -- the course being offered up by the swaperUser
+ * @property {[string]} createdDate -- Create date of this session
+ * @property {[string]} swapCompleted -- status of whether swapped or not
+ * @property {[string]} swapeeAccept -- status of swapee
+ * @property {[string]} swapeeUserId -- swapee user id
+ * @property {[string]} swaperUserId -- swaper user id
+ */
+
+/**
+ * @route POST /courses/swaps/accept
+ * @group Courses
+ * @param {SwapCourses.model} SwapCourses.body.required - A swap req contains userId and courseToGiveId, courseGetId, swapeeAccept
+ * @return {CourseSwapResponse.model} 201 - Accept successfully
+ * @return {ErrorResponse.model}  default - HttpError - Swap cannot created
+ * @security JWT
+ */
+
+module.exports = function (req, res, next) {
+  const swapReq = {
+    'userId': req.body.userId,
+    'courseToGiveId': req.body.courseToGiveId,
+    'courseToGetId': req.body.courseToGetId
+  }
+  return axios.post(urlJoin(DB_BASE_URL, 'courses', 'swap', 'accept'), swapReq)
```

Comment:
`swap` should be `swaps`
![image](https://user-images.githubusercontent.com/8997932/68420943-5b1d5980-016b-11ea-81c9-14ae94bc24e2.png)